src/os/ui/query/queryhandler.js

```diff
@@ -203,14 +203,15 @@ os.ui.query.QueryHandler.prototype.setSpatialRequired = function(value) {
 
 
 /**
+ * @param {os.query.BaseAreaManager?} areaManager
  * @param {!Object<string, string|boolean>} item
  * @return {boolean}
  */
-os.ui.query.QueryHandler.includes = function(item) {
+os.ui.query.QueryHandler.includes = function(areaManager, item) {
```

Comment:
This wouldn't be a breaking change if it were an optional second parameter. Same goes for other places this change was made.

Reply:
I did try that route, but it collided with the use of that function in `Array.filter` since the additional params didn't align